src/cli/vector_enrich_text.rs

```diff
@@ -167,7 +167,7 @@ fn vector_enrich_text(
     let start = SystemTime::now();
     for (i, batch) in batches.iter().enumerate() {
         let batch_start = SystemTime::now();
-        info!("Embedding batch {:?}/{} ", i + 1, batches.len());
+        println!("Embedding batch {:?}/{} ", i + 1, batches.len());
```

Comment:
I changed to println!() since these commands can take a little time I wanted the User to have an indicator of progress without running in info mode, since this log level is quite verbose.